scss/modules/_lists.scss

```diff
@@ -190,3 +207,26 @@
     margin-left: -$gutter-width * 3;
   }
 }
+
+/// Default list tick pattern styles
+/// @group Lists
+%list-ticks {
+  list-style: none;
+  margin-left: 0;
+
+  li {
+    background-repeat: no-repeat;
+    background-position: 0 .25em;
+    padding-left: 25px; 
+  }
+}
+
+// this is to get the variable inserted in an inline svg, we have to strip the # and replace with %23
+// it also checks to see if the color is a light color and adjusts the background color accordingly
+@function url-friendly-color($color) {
+  @if (lightness($color) > 50) {
```

Comment:
I think you should check that `$color` is a valid color so it doesnt bust, just errors.